google/cloud/firestore_v1/_helpers.py

```diff
@@ -222,7 +219,7 @@ def encode_dict(values_dict) -> dict:
     return {key: encode_value(value) for key, value in values_dict.items()}
 
 
-def reference_value_to_document(reference_value, client) -> DocumentReference:
+def reference_value_to_document(reference_value, client) -> Any:
```

Comment:
Ugh, we have cyclic imports here.  Could we use a string to get a better type hint?  E.g.:
```python
def reference_value_to_document(reference_value, client) -> "google.cloud.firestore_v1.document.DocumentReference":
```

Reply:
It still throws an error, for string also need to import the class.